### ISSUE TITLE
Transcribe Screen: use full width

### ISSUE THREAD

**Issue by benwbrum** [benwbrum/fromthepage#169]
The app is attractively styled, but this presents a problem for users trying to transcribe pages on large monitors.  Generally, they'd prefer to use as much of the screen as possible for the image and data entry, so the margins we have set are frustrating.

(more in screenshots to follow)

**Comment by benwbrum** [comment]
Compare the width of the page image viewing space in 
![old_transcribe_screenshot](https://cloud.githubusercontent.com/assets/199961/7443117/837b1830-f0fa-11e4-8152-05c80cc4f377.jpg) versus
![transcribe_screenshot](https://cloud.githubusercontent.com/assets/199961/7443118/8380d3ce-f0fa-11e4-854c-4e090ad6d6ae.jpg)

**Comment by kolking** [comment]
I've implemented the Fullscreen mode for the Transcribe page, so we can provide users with ability to use all width and height available for their display.

![image](https://cloud.githubusercontent.com/assets/4656448/7673510/4a61e266-fd1d-11e4-8e88-3359e88ef1fc.png)

Also, I've switched the panes to have the page image (zoomer) at the left and the edit area at the right. This is more natural for right-to-left languages. And I'm totally agree that it would be nice to also have an option to split pages horizontally.
